From 789afe30771dcaba3e99be55338a4a3f304ffc58 Mon Sep 17 00:00:00 2001
From: Raphael Mounier <mounierr07@gmail.com>
Date: Tue, 18 Oct 2022 21:37:22 +0200
Subject: [PATCH 13/13] NXP-NFC folder to force remounting partitions

Patch the AOSP system/core folder to force remounting partitions
---
 fs_mgr/fs_mgr_remount.cpp | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/fs_mgr/fs_mgr_remount.cpp b/fs_mgr/fs_mgr_remount.cpp
index deaf5f72d..99e89f7d1 100644
--- a/fs_mgr/fs_mgr_remount.cpp
+++ b/fs_mgr/fs_mgr_remount.cpp
@@ -417,12 +417,10 @@ static int do_remount(int argc, char* argv[]) {
         auto blk_device = entry.blk_device;
         auto mount_point = entry.mount_point;
 
-        auto found = false;
         for (auto it = mounts.rbegin(); it != mounts.rend(); ++it) {
             auto& rentry = *it;
             if (mount_point == rentry.mount_point) {
                 blk_device = rentry.blk_device;
-                found = true;
                 break;
             }
             // Find overlayfs mount point?
@@ -430,14 +428,9 @@ static int do_remount(int argc, char* argv[]) {
                 (mount_point == "/system" && rentry.mount_point == "/")) {
                 blk_device = rentry.blk_device;
                 mount_point = "/system";
-                found = true;
                 break;
             }
         }
-        if (!found) {
-            PLOG(INFO) << "skip unmounted partition dev:" << blk_device << " mnt:" << mount_point;
-            continue;
-        }
         if (blk_device == "/dev/root") {
             auto from_fstab = GetEntryForMountPoint(&fstab, mount_point);
             if (from_fstab) blk_device = from_fstab->blk_device;
-- 
2.25.1

